Dockerfile

```diff
@@ -1,20 +1,28 @@
-FROM python:3.12-slim-bookworm
+FROM python:3.12-slim-bookworm AS builder
 WORKDIR /tmp
 COPY . .
-RUN python -m pip install build --user && \
+RUN pip install --no-cache-dir build && \
     python -m build --sdist --wheel --outdir dist/
 
 FROM python:3.12-slim-bookworm
 
-COPY --from=0 /tmp/dist/*.whl /dist/
+LABEL maintainer="eduMFA <edumfa-dev@listserv.dfn.de>"
+LABEL version="2.2.0"
```

Comment:
Is there maybe an option to extract that somewhere? 

The metadata action already sets several labels:

```
            "Labels": {
                "org.opencontainers.image.created": "2024-07-17T05:53:42.254Z",
                "org.opencontainers.image.description": "Up-to-date multifactor authentication for academic institutions.",
                "org.opencontainers.image.licenses": "AGPL-3.0",
                "org.opencontainers.image.revision": "9124fd505cae556b5c35df46db00afe64d9fc8e6",
                "org.opencontainers.image.source": "https://github.com/eduMFA/eduMFA",
                "org.opencontainers.image.title": "eduMFA",
                "org.opencontainers.image.url": "https://github.com/eduMFA/eduMFA",
                "org.opencontainers.image.version": "2.2.0"
            }
```